spec/footer_spec.rb

```diff
@@ -21,7 +23,19 @@
 
   context "Verify Footer links functionalities", :LEAGUE76_step1 do
     it 'User is redirected to the "Shipping & Payment" informations page', :step_1 do
- 
```

Comment:
Before you make assertions, you need to perform the action as it asked on the test cases action columns, here is missing the action to click on Shipping and Payment link before the assertion call.
For tha above you will need:

- to instantiate FooterMethods class in order to use its methods in this class. 

- add the FooterMethods file in require_relative.

- call the click method of shipping and payment link, from FooterMethods class.